Source/Meadow.Foundation.Peripherals/Sensors.Distance.Me007ys/Samples/Me007ys_Sample/meadow.config.yaml

```diff
@@ -0,0 +1,2 @@
+﻿MonoControl:
+    Options: --jit
```

Comment:
You can remove this file if you want, JIT is enable by default now.

Reply:
will do

Reply:
actually ... I just leave it and we can do a bulk remove ... all of the samples have it currently